testsuite/MDAnalysisTests/coordinates/test_h5md.py

```diff
@@ -457,13 +457,23 @@ def test_step_from_frame(self, universe, Writer, outfile):
         for step, frame in zip(steps, frames):
             assert_equal(step, frame)
 
-    def test_has_setter(self, universe, Writer, outfile):
+    def test_has_property(self, universe, Writer, outfile):
         with Writer(outfile, universe.atoms.n_atoms) as W:
             W.write(universe)
             assert_equal(W.has_positions, W._has['position'])
             assert_equal(W.has_velocities, W._has['velocity'])
             assert_equal(W.has_forces, W._has['force'])
 
+    def test_has_setter(self, universe, Writer, outfile):
+        with Writer(outfile, universe.atoms.n_atoms) as W:
+            W.has_forces = False
+            W.has_velocities = False
+            W.write(universe)
+
+        uw = mda.Universe(TPR_xvf, outfile)
+        assert_equal(uw.trajectory.ts.has_positions, True)
+        assert_equal(uw.trajectory.ts.has_velocities, False)
+        assert_equal(uw.trajectory.ts.has_forces, False)
```

Comment:
Instead of `assert_equal` (which works for everything but particularly arrays) I would just use the simpl
```python
assert uw.trajectory.ts.has_positions is True
```
(for boolean, `is` is ok, but in general `==` is safer).

Reply:
Interesting facts:

```python
>>> np.bool(1) is True
True
>>> np.ones(1, dtype=bool)[0] is True
False
>>> np.ones(1, dtype=bool)[0] == True
True
```

;)

Reply:
Scary.

And odd:

```python
>>> isinstance(np.bool_, bool)
False
```

Reply:
Yeah the basic gist is that `np.bool_` is an actual type, as opposed to the Python bool which is a subclass of int. [There's a warning on the docs if you actively look for it.](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.bool_). It's just funny when `assert True is True` failing.

```python
>>> isinstance(True, int)
True
```

<img width="504" alt="Screen Shot 2021-08-01 at 1 49 18 pm" src="https://user-images.githubusercontent.com/31115101/130301557-5fe0a5d0-2c79-4d90-92dc-ef07771368ad.png">